pandas/core/frame.py

```diff
@@ -3705,7 +3705,7 @@ def transpose(
         nv.validate_transpose(args, {})
         # construct the args
 
-        dtypes = list(self.dtypes)
+        first_dtype = next(iter(self.dtypes), None)
```

Comment:
Can't this just be `self.dtypes[0] if self.dtypes else None`? Going through the iteration protocol seems overkill